tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
+    "WinObjC.Frameworks.UWP": "*",
+    "WinObjC.Frameworks.Core": "*",
+    "WinObjC.Language": "*",
+    "WinObjC.Logging": "*"
```

Comment:
>"*" [](start = 23, length = 3)

Should this be * or the last SDK release?
Same with project.json

I assume a user doesn't want to automatically upgrade to a new SDK without knowing it.